src/pydicom/pixels/utils.py

```diff
@@ -1274,7 +1275,8 @@ def _passes_version_check(package_name: str, minimum_version: tuple[int, ...]) -
         module = importlib.import_module(package_name, "__version__")
         return tuple(int(x) for x in module.__version__.split(".")) >= minimum_version
     except Exception as exc:
-        LOGGER.exception(exc)
+        if config.debugging:
+            LOGGER.warning(exc)
```

Comment:
Why not just use `LOGGER.debug()` here? Generally, I wonder why we need that `config.debugging` at all, if it just means the logging level is DEBUG.

Reply:
Mostly because you suggested warning 😛.

One thing I'd like to do for 3.1 is improve the overall logging.

Reply:
> Mostly because you suggested warning

Did I? Silly me...

> Technically config.debugging just enables extra logging (especially in frequently used code), not necessarily debug logging.

Hm, isn't that what log levels are for? We don't use info at all, I think...

> One thing I'd like to do for 3.1 is improve the overall logging.

🚀